src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,46 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
+        int booksCounter = 0;
+        Scanner scanner = new Scanner(System.in);
+        Book[] books = new Book[3];
+        System.out.println("Podaj informacje o trzech unikalnych ksiazkach");
+        while (booksCounter < books.length) {
+            Book book = createBook(scanner);
+            if (isUnique(book, books, booksCounter) != 0) {
+                books[booksCounter] = book;
+                booksCounter++;
+            }
+        }
+        for (Book book : books) {
+            System.out.println("Ksiazka: " + book.getTitle() + ", liczba stron: " + book.getPages());
+        }
+    }
+
+    private static Book createBook(Scanner scanner) {
+        System.out.println("Podaj tytul");
+        String title = scanner.nextLine();
+        System.out.println("Podaj ilosc stron");
+        int pages = scanner.nextInt();
+        Book book = new Book(title, pages);
+        return book;
+    }
 
+    private static int isUnique(Book book, Book[] books, int booksCounter) {
+        int innerMethodCounter = 0;
+        for (int i = 0; i <= booksCounter; i++) {
+            if (book.getTitle().equals(books[i].getTitle())) {
```

Comment:
Tutaj powinieneś używać metody equals z klasy Book (której aktualnie nie masz). Czyli możemy mieć dwie książki o takim samym tytule, ale z różną liczbą stron - traktujemy je wtedy jako różne książki 
(standardowo lepiej by było dodać do książki jeszcze np. numer isbn, wtedy jest to bardziej oczywiste, że tytuły książek się powtarzają, ale sam tytuł nie świadczy o unikalności)